docs/source/cli/notebooks/Notebook_4_MT_CLI.ipynb

```diff
@@ -0,0 +1,312 @@
+    "!pip install dgl==1.1.3 -f https://data.dgl.ai/wheels-internal/repo.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76f8f9c9",
+   "metadata": {},
+   "source": [
+    "### 1. Create the example ACM graph data\n",
+    "---\n",
+    "This notebook uses the ACM graph in the example. We use the following script to create the ACM graph data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e3ae0728",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir example\n",
+    "!wget -O /example/acm_data.py https://github.com/awslabs/graphstorm/raw/main/examples/acm_data.py\n",
+    "!python /example/acm_data.py --output-path /example/acm_raw"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e524e0c6",
+   "metadata": {},
+   "source": [
+    "The ACM graph data includes node files and edge files. It also includes a JSON configuration file describing how to construct a graph for model training. More details can be found in [Use Your Own Data (ACM data example)](https://graphstorm.readthedocs.io/en/latest/tutorials/own-data.html#use-your-own-data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "2c3e63a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ls -al /example/acm_raw/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee3ca793",
+   "metadata": {},
+   "source": [
+    "### 2. Construct and Partition ACM Graph\n",
+    "---\n",
+    "Since GraphStorm is designed naturally for distributed GNN training, we need to construct a graph and split it into multiple partitions. In this example, for simplicity, we create a graph with one partition (no actual splitting)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8602b955",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m graphstorm.gconstruct.construct_graph \\\n",
+    "           --conf-file /example/acm_raw/config.json \\\n",
+    "           --output-dir /example/acm_gs \\\n",
+    "           --num-parts 1 \\\n",
+    "           --graph-name acm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de68edcf",
+   "metadata": {},
+   "source": [
+    "The generated ACM graph contains all the information required for GNN model training. For more details of preparing data for multi-task learning, please refer to [Preparing multi-task learning data](https://graphstorm.readthedocs.io/en/latest/advanced/multi-task-learning.html#preparing-the-training-data)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5735fdf1",
+   "metadata": {},
+   "source": [
+    "### 3. GNN Model Training \n",
+    "---\n",
+    "\n",
+    "Once the graph constucted, we can call GraphStorm multi-task learning CLI to run model training. Before kicking off the model training, we need to create a YAML configuration file for the CLI."
```

Comment:
```suggestion
    "Once the graph constucted, we can call GraphStorm multi-task learning CLIs to run model training. Before kicking off the model training, we need to create a YAML configuration file for the CLI."
```

Reply:
This is a single CLI command. I changed it into `call the GraphStorm multi-task learning CLI to run`